rfc/guides-comments.md

```diff
@@ -0,0 +1,126 @@
+
+## RFC for Adding Comments to our Deployment Guides
+
+  
+
+Options available for adding comments to the deployment guides are as follows:
+
+  
+1. Disqus: <https://disqus.com/>
+
+Really good third party service and the most popular. It has 3 options:
+
+- Basic: This has ad's (not recommended)
+
+- Plus $9 per month (Suitable for under 50k visitors daily)
+
+- Pro $89 per month (Suitable for 150k visitors daily)
+
+  
+
+Pros:
+
+- It provides a dashboard to manage for all the comments posted.
+
+- Analytics available for comments on the dashboard.
+
+- We can control the look and feel of the design
+
+- Very easy to install
+
+
+Cons
+
+- Being a third party service, we don't know what our comments become and how disqus uses the data retrieved from the comments
+
+- Disqus doesn't have a great track record on privacy
+
+- Does not allow anonymous content
+
+- It adds a lot of page weight. About 450kB of data
+
+  
+
+1. Just comments [https://just-comments.com/](https://just-comments.com/pricing.html)
+
+Also a third party service and very similar to Disqus
+
+  
+Pros:
+
+- Look and feel of the design is customizable
+
+- Pay as you go but you will need to pay 5.30 dollars for having an active api key per month
+
+- Page weight is about 25kb
+
+- They do not track users
+
+  
+
+Cons
```

Comment:
Same issue here in terms of JS/AJAX/SEO.

Reply:
Comments are also indexed by google